server/block/cactus.go

```diff
@@ -0,0 +1,69 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/particle"
+	"math/rand"
+)
+
+// A Cactus is a naturally occurring block found in deserts
+type Cactus struct {
+	solid
+	snare
+
+	// Age is the groth state of cactus. values from 0 to 15 maybe?
+	Age int
+}
+
+// FlammabilityInfo ...
+func (c Cactus) FlammabilityInfo() FlammabilityInfo {
+	return newFlammabilityInfo(5, 5, true)
+}
+
```

Comment:
```suggestion
```

Cactus blocks are not flammable

Reply:
I'll knock this out, but is there a good resource to find the properties like this?

Reply:
The wiki usually has some solid info on this stuff,